BeagleBone_black/src/utils/concurrent/Thread.hpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Authors: M. Kristien
+ * Organisation: HYPED
+ * Date: 21. February 2018
+ * Description:
+ *
+ *    Copyright 2018 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef BEAGLEBONE_BLACK_UTILS_CONCURRENT_THREAD_HPP_
+#define BEAGLEBONE_BLACK_UTILS_CONCURRENT_THREAD_HPP_
+
+#include <cstdint>
+
+// Forward declaration
+namespace std { class thread; }
```

Comment:
Does this mean that whoever is including this file is also expected to do `#include <thread>`? Why is that desirable?

Reply:
no, that is not necessary,, it is only included in `main_threading.cpp` because that main also demonstrates the use of `std::thread`